### PR TITLE
[minor] Sorting the contents of core.toml

### PR DESCRIPTION
**Pull request by phrawzty**
Personally I think the sort order should be randomised every time we push the site, but hey…

### CODE REVIEW

File: data/core.toml [hunk 1]
```diff
@@ -16,27 +16,27 @@ advisory = [
 "Serhat Can",
 "Jennifer Davis",
 "Patrick Debois (founder, 2009–2014 lead)",
+"Yvo van Doorn (2021–2023 co-chair)",
 "Nathen Harvey",
 "Bridget Kromhout (2015–2020 lead)",
 "Ken Mugrage",
 "Matty Stratton (2021–2022 co-chair)",
```
Comment (adrianmoisey) [inline data/core.toml:23]: Heh, this trailing comma is an issue
```suggestion
"Matty Stratton (2021–2022 co-chair)"
```

File: data/core.toml [hunk 2]
```diff
@@ -16,27 +16,27 @@ advisory = [
...
+"Gildas Le Nadan",
 "Stephen Nelson-Smith",
+"Mike Rosado"
+"Andrew Clay Shafer",
 "Julian Simpson",
 "Christian Trabold",
 "John Vincent",
 "James Wickett",
-"Andrew Clay Shafer",
 "John Willis",
```
Comment (adrianmoisey) [inline data/core.toml:41]: This trailing comma is also an issue

```suggestion
"John Willis"
```

Comment (phrawzty) [reply data/core.toml:41]: I don't think trailing commas matter in TOML?

File: data/core.toml (hunk 1 above)
Comment (phrawzty) [reply data/core.toml:23]: Last-line-of-a-list commas matter don't matter in TOML:

https://toml.io/en/v1.0.0#array

File: data/core.toml (hunk 2 above)
Comment (phrawzty) [reply data/core.toml:41]: Missing commas definitely matter though, yes ;)

File: data/core.toml (hunk 1 above)
Comment (adrianmoisey) [reply data/core.toml:23]: Huh, that's neat. 
I'm not sure I understand why it failed as we've had TOML without the trailing comma

Your solution of adding a trailing comma to all the arrays is a good one

**Pull request merged**